integration/angular/simple-message.ts

```diff
@@ -81,3 +80,7 @@ if (util.Long !== Long) {
   util.Long = Long as any;
   configure();
 }
+
+function isSet(value: any): boolean {
+  return value !== null && value !== undefined;
```

Comment:
Now that this is pulled out into a nicely named helper, you can even use `value != null`.

Do we need type narrowing? You can do something like

```
function isSet<X>(value: X  | undefined | null): value is X {
  return value != null;
}
```